Cartoon element

rebecca-thompson opened the pull request:
## Why are you doing this?

This adds support in the cartoon element, a multi-image element which displays different sets of images (which roughly correspond to cartoon panels) at different breakpoints.

Cartoon elements are only permitted in the Picture content type and come through in the main media, so support for this element has been limited to that scenario only.

The majority of the rendering logic resides in DCR which can be seen in the companion pr here: https://github.com/guardian/dotcom-rendering/pull/9416

## Changes

- Bumping CAPI libs to include Cartoon element definition
- Add `cartoonImages` to the fronts - this allows us to use the lightbox feature

## Screenshots

https://github.com/guardian/editions/assets/33927854/5727405b-a769-4e6d-b3f4-2ddd51baed5e

jimhunty commented on projects/archiver/src/tasks/front/helpers/media.ts:
```diff
@@ -44,12 +44,16 @@ export const getImagesFromArticle = (
     console.log('Images are: ' + JSON.stringify(images))
 
+    const cartoonImages = article.cartoonImages || []
+    console.log('Cartoon images are: ' + JSON.stringify(cartoonImages))
```
Please remove

rebecca-thompson replied:
I assume you meant the log line? I have removed it here: https://github.com/guardian/editions/pull/2337/commits/aaae0fdcb3bc695ef43b185129cb5f07b4ef2ba6. The other images are logged out in the lines above, so this was done to maintain consistency.

jimhunty reviewed (approved):
All looks fine to me, just the one change.

I would like to see some unit tests, but this project doesn't require them.

rebecca-thompson commented:
> All looks fine to me, just the one change.
> 
> I would like to see some unit tests, but this project doesn't require them.

This is a really good point. As the functionality mostly resides in the image picker, I've added some tests for that here: https://github.com/guardian/editions/pull/2337/commits/2df9440f84a053cb9a221d216e0f1fe2d44cc74e. This highlighted some missing behaviour where the images should fallback to desktop images if no mobile images are present, so I have added that in also
